lib/guardian_db.ex

```diff
@@ -154,5 +148,23 @@ defmodule GuardianDb do
     end
   end
 
-  def repo, do: @repo
+  def repo do
+    :guardian_db
+    |> Application.get_env(GuardianDb, [])
+    |> Keyword.get(:repo)
+    |> case do
```

Comment:
Please do not pipe into a case statement 😰  Could you please make this into a function and pipe into that?